src/platform/graphics/compute.js

```diff
@@ -55,10 +58,12 @@ class Compute {
      * @param {import('./graphics-device.js').GraphicsDevice} graphicsDevice -
      * The graphics device.
      * @param {import('./shader.js').Shader} shader - The compute shader.
+     * @param {string} [name] - The name of the compute instance, used for debugging only.
      */
-    constructor(graphicsDevice, shader) {
+    constructor(graphicsDevice, shader, name = 'Noname') {
```

Comment:
Maybe `Unnamed` would be better. We already use that for GraphNodes.

Reply:
GraphNode uses `Untitled`, and that looks strange in logging where I log names. Happy with `Unnamed`.